tutorial/utlis_tutorial.py

```diff
@@ -0,0 +1,76 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+plt.rcParams['figure.dpi'] = 200  # figure dpi
+
+
+# animation
+def update(num, lines, points, coords):
+    start = max(0, num - 100)
+    outputs = []
+    for i in range(len(lines)):
+        lines[i].set_data(coords[i]['x'][start:num], coords[i]['y'][start:num])
+        points[i].set_data(coords[i]['x'][num], coords[i]['y'][num])
+        outputs.append(lines[i])
+        outputs.append(points[i])
+    return outputs
+
+
+def plotEnergies(energies, ax, t, equal=False, name=None):
+    if equal:
+        ax.plot(t, energies)
+        ax.axis("equal")
+    elif name is not None:
+        ax.plot(t, energies, label=name)
+        ax.legend(loc="upper right")
+    else:
+        ax.plot(t, energies)
+
+
+def plotVelo(velo, ax, fig, name="", equal=None):
```

Comment:
Could you respect the snake variable convention here?
You can check code style by invoking `pylint tutorial/utlis_tutorial.py` from the root of this repository.

Reply:
Thanks! I reformat the codes according to the pylint instruction.